.github/workflows/publish_doc.yml

```diff
@@ -38,12 +38,6 @@ jobs:
       - name: Conda Setup
         uses: "./.github/actions/step_create_conda_environment"
 
-      - name: Display environment variables
```

Comment:
Why removed this?

Reply:
Was initially copied from other pipeline, then in the original place Philip said it's for the debug usage when he first added this and suggested to remove.